test/test-suite-1/test.default-index.js

```diff
@@ -218,6 +244,34 @@ module.exports = function (dbType, context) {
       }).then(function (resp) {
         resp.should.deep.equal({
           warning: 'no matching index found, create an index to optimize query time',
+          docs: []
+        });
+      });
+    });
+  });
+
+  it.skip('empty selector returns empty docs', function () {
```

Comment:
Yes I did mean to add this. I was going to fix it but I'm not sure if the issue is pouchdb-find or Mango. So I'm skipping it for now.